pipelines/rj_segur__forca_municipal/task.py

```
@@ -0,0 +1,406 @@
+            params: Parâmetros da query (opcional)
+            client: Cliente httpx reutilizável (opcional)
+
+        Returns:
+            Resposta JSON ou None em caso de erro
+        """
+        if not self.access_token:
+            print("Não autenticado. Execute authenticate() primeiro.")
+            return None
+
+        url = f"{self.base_url}/{endpoint.lstrip('/')}"
+        breakpoint()
```

Ruff (0.15.12)

[error] Trace found: `breakpoint` used

(T100)